lib/ash/type/enum.ex

```diff
@@ -118,6 +191,12 @@ defmodule Ash.Type.Enum do
   @doc "finds the valid value that matches a given input term"
   @callback match(term) :: {:ok, atom} | :error
 
+  @optional_callbacks [
```

Comment:
Hm...this is confusing to me. I don't think these are optional callbacks, they are *overridable* callbacks.

Reply:
After thinking more about it, it's probably better to revert the callback changes and just remove the reference of `gettext` from the docs.  Users could always define a `label/2`, `description/2`, `details/2` themselves without the need for the optional callbacks and achieve the same thing.

Reply:
But the callbacks need to be marked as overridable so that the y can define their own.